lib/DxcSupport/HLSLOptions.cpp

```diff
@@ -336,15 +337,15 @@ int ReadDxcOpts(const OptTable *optionTable, unsigned flagsToInclude,
       opts.OptLevel = 2;
     if (A->getOption().matches(OPT_O3))
       opts.OptLevel = 3;
+    if (A->getOption().matches(OPT_Od)) {
+      opts.DisableOptimizations = true;
```

Comment:
You should set disable optimatsons to false for non Od flag, right?

Reply:
It is set to false at line 330.

Reply:
Ah, ok missed that.